### ISSUE TITLE
jpx image render problem

### ISSUE THREAD

**Issue by proudlife** [mozilla/pdf.js#12738]
version: 2.5.207
[1版-9-8-2.pdf](https://github.com/mozilla/pdf.js/files/5694004/1.-9-8-2.pdf)

**Comment by timvandermeij** [comment]
From the console:

```
Warning: Optional content group not found: null util.js:358:4
Warning: Knockout groups not supported. util.js:358:4
```

**Comment by Snuffleupagus** [comment]
WFM in Firefox 94 with its built-in PDF Viewer (version `2.11.243`); most likely fixed by PR #13698.
